core/src/main/java/nl/nn/adapterframework/filesystem/IMailFileSystem.java

```diff
@@ -25,25 +25,21 @@
 	public String MAIL_MESSAGE_ID = "Message-ID";
 	
 	public Iterable<String> getToRecipients(M emailMessage) throws FileSystemException ;
-
 	public Iterable<String> getCCRecipients(M emailMessage) throws FileSystemException;
-	
 	public Iterable<String> getBCCRecipients(M emailMessage) throws FileSystemException;
 	
 	public String getFrom(M emailMessage) throws FileSystemException;
+	public String getSender(M emailMessage) throws FileSystemException;
```

Comment:
Ik zou het fijn vinden als er een regel commentaar bij staat wat het verschil tussen een `from` en een `sender` is...